cups/translate-attrs.go

```diff
@@ -67,6 +67,16 @@ func getUUID(printerTags map[string][]string) string {
 }
 
 func getState(printerTags map[string][]string) cdd.CloudDeviceStateType {
+	// offline-report reason is set when printer is offline (e.g. USB cable is disconnected)
```

Comment:
`printer-state` stays `idle` and `printer-is-accepting-jobs` stays `true`. That's the main reason for this workaround. I checked with lpoptions and the only thing that changes is `printer-state-reasons`.
Quick look at CUPS [sources](https://github.com/apple/cups/blob/68a36223d03dd2dddfc5e6422aa2126b4e987656/backend/usb-darwin.c) makes this statement more convincing (at least on OS X).